interactions/lewd/lewd_interactions.dm

```diff
@@ -79,6 +80,17 @@
 				to_chat(user, "<span class='warning'>They're still exhausted from the last time. They need to wait [DisplayTimeText(target.refactory_period * 10, TRUE)] until you can do that!</span>")
 			return FALSE
 
+		if(require_ooc_consent)
```

Comment:
As gross as this is to write

1.
should also check consent on self
there are various damaging interactions, and a player could make themselves 'immune' whilst still damaging others.

2.
so as to not make this a spammable thing, may wish to tie this to the 
`user_not_tired && user.refactory_period` flags similar to 
https://github.com/BadDeathclaw/TG-Claw/blob/be7247d0a7161f5747b66ab95b4430656feca691/interactions/lewd/lewd_interactions.dm#L37
Otherwise (assuming 1 is complete) Tim may consent, damage another player, and quickly turn off consent before the player can 'retaliate'.

Reply:
Made the change suggested in 1, both should need to have set it to on for *any* interaction to take place.

Reply:
Fixed 2.